lib/sle2docker/cli.rb

```diff
@@ -47,7 +47,7 @@ def version
     private
 
     def ensure_can_access_dockerd
-      output = `docker info`
+      output = `docker info 2>/dev/null`
```

Comment:
The reason I included this is because a part of problem #16 is the message "WARNING: No swap limit support" which is generated by the docker info command. This occurs on most VMs since Docker is unable to enforce swappiness parameters on its containers in those situations. Since it's unrelated to whether we can actually access dockerd I figured simply redirecting it to /dev/null wouldn't be a problem. Would you prefer we just ignore the message? (It seems to cause a lot of confusion though since it will pop up for most all sle2docker commands since they all frequently use the ensure_can_access_dockerd function.) Or did you have something else in mind?